livekit-plugins/livekit-plugins-turn-detector/livekit/plugins/turn_detector/__init__.py

```diff
@@ -27,11 +27,12 @@ def __init__(self):
         super().__init__(__name__, __version__, __package__, logger)
 
     def download_files(self) -> None:
-        from transformers import AutoModelForCausalLM, AutoTokenizer
+        from optimum.onnxruntime import ORTModelForCausalLM
```

Comment:
Any way to remove the optimum dependency so we can get rid of torch? 
https://pypistats.org/packages/optimum

Reply:
i'm having some trouble getting the plain onnxruntime working so i figured we could do this in the meantime. i'll make a follow up PR that removes pytorch next week.